rules/integrations/o365/collection_microsoft_365_new_inbox_rule.toml

```diff
@@ -43,13 +43,8 @@ type = "query"
 
 query = '''
 event.dataset:o365.audit and event.provider:Exchange and
-event.category:web and event.action:("New-InboxRule" or "Set-InboxRule") and
-    (
-        o365.audit.Parameters.ForwardTo:* or
```

Comment:
I wouldn’t remove all of these conditions since they’re here to catch exfililtration and such. 

Removing the conditions here can generate a lot of noise in large environments as it will trigger in innocuous activity like sending to a folder in one’s own mailbox or flagging an email for follow up.